common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -607,6 +608,9 @@ def _student_or_public_view(self, context, prereq_met, prereq_meta_info, banner_
 
         fragment = Fragment()
         params = self._get_render_metadata(context, display_items, prereq_met, prereq_meta_info, banner_text, view, fragment)  # lint-amnesty, pylint: disable=line-too-long
+        if settings.FEATURES.get('SHOW_PROGRESS_BAR', False) and getattr(settings, 'COMPLETION_AGGREGATOR_URL', ''):
```

Comment:
Please update this to use a settings toggle (see https://github.com/openedx/edx-toggles/blob/98960e4793e4af47c5f4144efcf418d0e96c8685/docs/decisions/0003-django-setting-toggles.rst )

Reply:
@rgraber Thanks I wasn't aware of that. I've updated it now. Does it look ok?